ts/Extensions/Annotations/Popup.ts

```diff
@@ -1019,25 +1019,22 @@ H.Popup.prototype = {
             selectBox.setAttribute('id', PREFIX + 'select-' + optionName);
 
             // list all series which have id - mandatory for creating indicator
-            chart.series.forEach(function (serie): void {
+            chart.series.forEach(function (series): void {
+                seriesOptions = series.options as any;
```

Comment:
You are casting the whole `seriesOptions` variable to `any`, but only `seriesOptions.params` is throwing an error (other `seriesOptions.name` or `seriesOptions.id` are fine.

I would think about how we can get rid of `as any` totally or at least cast to `any` only `options.params` , not the whole variable (like previously, before your changes).